### DISCUSSION TITLE
Give a temporal access to a client

### DISCUSSION THREAD

**Discussion by franckarts** [LizardByte/community#177] [Sunshine]
Hi,

Thanks for this awesome tool, it works amazingly well!

Is there any way to give temporal access to a particular user i.e. when he connects to sunshine, we get a message asking the permission to share the screen ? I use sunshine to play with friends but giving permanent access to my computer is a problem. On the other hand, I still need to run sunshine as a service for local streaming to my TV.

Thanks for the advices!

**Comment by ReenigneArcher** [comment]
This is not currently possible with Sunshine.

**Reply by franckarts** [reply]
Ok, thank you for the answer. I am moving this topic to the "Feature  Requests" category and hope that this is something that could interest users / authors :) .
